packages/node/src/lib/types.ts

```diff
@@ -0,0 +1,56 @@
+import { HTTPClient } from './http-client'
+
+export type OAuthSettings = {
+  /**
+   * The OAuth App ID from Access Management under Workspace Settings in the Segment Dashboard.
+   */
+  clientId: string
+  /**
+   * The private key that matches the public key set in the OAuth app in the Segment Dashboard.
+   */
+  clientKey: Buffer
```

Comment:
Buffer type is potentially coming from the ambient node environment-- so I don't think we really want to use this type? cc @chrisradek

Reply:
Fixed